added short intros to section landing pages (#313)
elixir-europe/infectious-diseases-toolkit: tools-and-resources-list/index.md

datatable: true
---

>  To effectively address the complexities of infectious diseases, professionals can browse all tools and resources mentioned in an infectious diseases toolkit. This approach ensures that stakeholders are well-equipped with reliable and proven tools to combat infectious diseases.

Reviewer: ```suggestion
This page captures all tools and resources mentioned across the Infectious Diseases Toolkit. 
```
